File: SecondYear/seeeyaa_PariGupta_2226CSE(AI)1096_2/Week_2/CheckPalindrome.cpp
bool palindrome(int n)
{
    // Write your code here
    int n1=n,rem,revnum=0;
    while(n>0)
    {
        rem=n%10;
        revnum=revnum*10+rem;
        n/=10;
    }
    if(n1==revnum) return 1;
    return 0;

    
}

File: SecondYear/aayushker_AayushkerSingh_2226CSE(AI)1010_2/Week_2/Check_Palindrome.py
n=int(input())  
# taking n as a input 
## write your code !!
a=n
s=0
rem=0
while(n!=0):
    rem=n%10
    s =s*(10)+rem
    n=n//10
if(a == s):
    print("true")
else:
    print("false")   